salt/gis.sls

```diff
@@ -0,0 +1,127 @@
+# get the software packages installed
+base:
+  pkg.installed:
+    - install_recommends: false
+    - pkgs:
+      # just some basic packages, not specific to the carto setup.
+      - git 
+      - unzip
+      - curl
+      - build-essential
+      - software-properties-common
+    - refresh: true
+
+# installing the required GIS-software, plus...
+# create the postgresql database and enabling PostGIS
+gis:
+  pkg.installed:
+    - install_recommends: false
+    - pkgs:
+      # here come the goodies; PostGIS is elementary to a current dev. environment.
+      - postgresql-9.3-postgis-2.1
+      - postgresql-contrib-9.3
+      - proj-bin
+      - libgeos-dev
+  # Let's make it easy for our GIS-in-aid software to access our PostGIS database
+  file.blockreplace:
+    - name: /etc/postgresql/9.3/main/pg_hba.conf
+    - marker_start: "# TYPE  DATABASE"
+    - marker_end: '# "local" is'
+    - content: "local   all             gisuser                                 trust"
```

Comment:
@pnorman I believe so, unless I give the `gisuser` a password and tell it to connect using TCP/IP, instead of a local Unix socket. A local Unix socket should give better transfer rates, so unless the user specifies otherwise in the Vagrantfile for what database and host to use, it should probably remain, unless there are other concerns.